docker-compose-setup/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+services:
+  coatrack-postgres:
+    image: postgres:9.4.26
+    container_name: coatrack-postgres
+    volumes:
+      - coatrack-postgres:/var/lib/postgresql/data
```

Comment:
```suggestion
      - coatrack-postgres-data-storage-volume:/var/lib/postgresql/data
```

Reply:
Done, but I preferred 'coatrack-database-volume' which is clear and short, imo.